pkg/vim

```diff
@@ -1,11 +1,12 @@
 [mirrors]
-ftp://ftp.vim.org/pub/vim/unix/vim-8.0.586.tar.bz2
+ftp://ftp.vim.org/pub/vim/unix/vim-8.1.tar.bz2
 
 [vars]
-filesize=10866752
-sha512=f8f7f9babf7bca50bb5e80b85ca3d74172c3da783b3a2b13f67cce802f01974fe74042b67dda4de727bf2cf2c1399e3443428b14fd6b90f79be98a88ea03d621
-pkgver=4
-tardir=vim80
+filesize=11258868
+sha512=141074bdc25d68f7ced9b596da3b5c6e719f9e316b3ce6cb30d0c501d0c5396e2c6d0efc6425a41f2320cd80593fbb
+1c5ca64fb5e2178f1fb16cdbb470aa630e
```

Comment:
that sha512 hash seems bogus

Reply:
it had an extra new line, thanks. fixed.

Reply:
but it also seems that butch does not check the sha512 before building a pkg.

Reply:
butch download does the checks. it's assumed that once a tarball is downloaded and verified, it's good to use.